package.json

```diff
@@ -17,17 +17,18 @@
     "eslint-config-prettier": "^6.0.0",
     "eslint-plugin-node": "^9.1.0",
     "eslint-plugin-prettier": "^3.1.0",
-    "execa": "^2.0.3",
     "jest": "^24.8.0",
     "prettier": "^1.18.2",
     "release-it": "^12.3.4",
     "release-it-lerna-changelog": "^1.0.3"
   },
   "dependencies": {
+    "execa": "^2.0.4",
     "fs-extra": "^8.1.0",
     "git-repo-info": "^2.1.0",
     "puppeteer": "^1.19.0",
     "sync-disk-cache": "^1.3.3",
+    "tree-kill": "^1.2.1",
```

Comment:
Seems like this is a test specific dep, can we move it to decades?

Reply:
Sure.

Reply:
Decades?